src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -265,6 +265,7 @@
     <xsd:complexType name="cache_pool">
         <xsd:attribute name="name" type="xsd:string" use="required" />
         <xsd:attribute name="adapter" type="xsd:string" />
+        <xsd:attribute name="tags" type="xsd:string" />
```

Comment:
isn't it actually a boolean node ?

Reply:
Hm, it can indeed be a boolean. But also a service ID: https://github.com/symfony/symfony/pull/26934#issuecomment-381610712

Reply:
ah, I forgot that